tests/workflow_tests/test_deployment_update.py

```diff
@@ -2004,3 +2012,104 @@ def test_remove_deployment(self):
         self.assertEquals(len(deployment_update_list), len(undel_depups_ids))
         for i in deployment_update_list.items:
             self.assertIn(i['id'], undel_depups_ids)
+
+
+class TestDeploymentUpdateMisc(DeploymentUpdateBase):
```

Comment:
ok, i moved all but two tests to my new class; kinda difficult seeing that on github comparison view but you can try and just view the file itself